gridcapa-core-valid-app/src/main/java/com/farao_community/farao/gridcapa_core_valid/app/study_point/StudyPointService.java

```diff
@@ -58,9 +62,16 @@ public StudyPointResult computeStudyPoint(StudyPoint studyPoint, Network network
             Map<String, InitGenerator> initGenerators = setPminPmaxToDefaultValue(network, scalableZonalData);
             NetPositionsHandler.shiftNetPositionToStudyPoint(network, studyPoint, scalableZonalData, coreNetPositions);
             resetInitialPminPmax(network, scalableZonalData, initGenerators);
-            String url = saveShiftedCgm(network, studyPoint);
+            String shiftedCgmUrl = saveShiftedCgm(network, studyPoint);
+            result.setShiftedCgmUrl(shiftedCgmUrl);
+            String raoRequestId = String.format("%s-%s", network.getNameOrId(), studyPoint.getId());
+            RaoResponse raoResponse = startRao(raoRequestId, shiftedCgmUrl, jsonCracUrl, raoParametersUrl);
             result.setStatus(StudyPointResult.Status.SUCCESS);
-            result.setShiftedCgmUrl(url);
+            result.setNetworkWithPraUrl(raoResponse.getNetworkWithPraFileUrl());
+            result.setRaoResultFileUrl(raoResponse.getRaoResultFileUrl());
```

Comment:
I would group these two lines into a new method result.updateAfterRao(raoResponse)

Reply:
I'm not sure that it is necessary... We can discuss it